### ISSUE TITLE
"Please restart the app and disable antivirus" error

### ISSUE THREAD

**Issue by ethrea** [jely2002/youtube-dl-gui#362]
I get the message "Please restart the app and disable antivirus." when trying to put in any link. Message persists even when any anti virus & firewall is off. Didn't have the problem before, but just got a fresh install of windows on a new pc, so could be something in windows hiding away.

![Open_Video_Downloader_UTcH9g3mVP](https://user-images.githubusercontent.com/95581328/169218761-4fb21bde-e8c9-449a-9676-d62b15080bf2.png)

**Comment by anothermelon** [comment]
having the same issue!

**Comment by ericmp33** [comment]
Having the same issue here!
![image](https://user-images.githubusercontent.com/60661635/169663992-7468d0a0-38c0-42fb-bae1-3dc1c36f21b9.png)

And yeah, I tried to restart the app and disable my antivirus several times. Also tried restarting the computer and reinstalling the program :/

**Comment by ArchemedIan** [comment]
same issue

**Comment by FR46M3N7-P4R71CL3** [comment]
### I found a solution:

1. First, download the YT-DLP Binary from the following address:
`https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp.exe`

2. Open Run (Windows Key + R) and in the dialog box, paste the below:
`C:\Users\%username%\AppData\Local\Programs\youtube-dl-gui\resources\app.asar.unpacked\binaries`
and then hit **Enter** key or **OK** Button.

3. Now navigate to the YT-DLP download folder and copy the downloaded file (from Step 1) and Paste the file into the folder we just opened in Step 2. Make sure the filename stays/is "yt-dlp"(.exe)

### Restart the application and now it should work! 🚀

For some reason, a fresh installation doesn't seem to install this binary and the update check too fails to observe that the file is missing.

cc: @ericmp33 @ArchemedIan @anothermelon @ethrea @itscaro

**Comment by wuwendell** [comment]
Yep, it just seems like that yt-dlp.exe is not downloaded on a fresh install.

**Comment by richardsondev** [comment]
Thank you @FR46M3N7-P4R71CL3 for sharing the solution to get it running again!

I took a deeper look and it appears GitHub made some changes recently in the release download flow. This has broken the automatic download of `yt-dlp` here. I have opened a PR to update the logic: #375

**Comment by nguyenvinhlinh** [comment]
Thank you!

**Comment by JukeStuff** [comment]
Great This worked :)

**Comment by MM12300** [comment]
@FR46M3N7-P4R71CL3 procedure working but then Error? Unhandled error.
Missing ffmpeg.

**Comment by FR46M3N7-P4R71CL3** [comment]
> @FR46M3N7-P4R71CL3 procedure working but then Error? Unhandled error. Missing ffmpeg.

Try reinstalling the app? Make sure you have ffmpeg present in install directory or in any subdirectory.
I don't use this app anymore so I'm not sure if something's broken.

**Comment by guyster104** [comment]
I have a question. I have the open video downloader portable exe. Do I need to create the folders mentioned in the solution, or where can I put the youtube-dlp binary for it to work? I don't see anywhere in settings to specify the path. Thanks in advance for your help.

**Comment by guyster104** [comment]
Also, is 2.4.0 still newest version?

**Comment by himyjan** [comment]
> ### I found a solution:
> 1. First, download the YT-DLP Binary from the following address:
>    `https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp.exe`
> 2. Open Run (Windows Key + R) and in the dialog box, paste the below:
>    `C:\Users\%username%\AppData\Local\Programs\youtube-dl-gui\resources\app.asar.unpacked\binaries`
>    and then hit **Enter** key or **OK** Button.
> 3. Now navigate to the YT-DLP download folder and copy the downloaded file (from Step 1) and Paste the file into the folder we just opened in Step 2. Make sure the filename stays/is "yt-dlp"(.exe)
> 
> ### Restart the application and now it should work! 🚀
> For some reason, a fresh installation doesn't seem to install this binary and the update check too fails to observe that the file is missing.
> 
> cc: @ericmp33 @ArchemedIan @anothermelon @ethrea @itscaro

https://github.com/StefanLobbenmeier/youtube-dl-gui/releases

or

command line version
https://github.com/jely2002/youtube-dl-gui/issues/354#issuecomment-1127269708

**Comment by cuhsy** [comment]
> ### I found a solution:
> 
>     1. First, download the YT-DLP Binary from the following address:
>        `https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp.exe`
> 
>     2. Open Run (Windows Key + R) and in the dialog box, paste the below:
>        `C:\Users\%username%\AppData\Local\Programs\youtube-dl-gui\resources\app.asar.unpacked\binaries`
>        and then hit **Enter** key or **OK** Button.
> 
>     3. Now navigate to the YT-DLP download folder and copy the downloaded file (from Step 1) and Paste the file into the folder we just opened in Step 2. Make sure the filename stays/is "yt-dlp"(.exe)
> 
> 
> ### Restart the application and now it should work! 🚀
> 
> For some reason, a fresh installation doesn't seem to install this binary and the update check too fails to observe that the file is missing.
> 
> cc: [@ericmp33](https://github.com/ericmp33) [@ArchemedIan](https://github.com/ArchemedIan) [@anothermelon](https://github.com/anothermelon) [@ethrea](https://github.com/ethrea) [@itscaro](https://github.com/itscaro)

Thank you for everything 🙏

**Comment by ErikBrown2** [comment]
You can also use this yt-dlp updater. Just unzip it to any folder and double click it. It will then download the latest version of yt-dlp.exe and put it in the right directory for Open Video Downloader.

[yt-dlp_Updater.zip](https://github.com/user-attachments/files/18466898/yt-dlp_Updater.zip)

**Comment by thiwankarandeni** [comment]
Thank you @FR46M3N7-P4R71CL3, it works perfectly now.

**Comment by jely2002** [comment]
Hey all, this issue has been fixed in v3. As such I'll be closing this issue.
